app-front/src/app/activite/tabs/TabExport.tsx

```
@@ -168,10 +168,7 @@ export default function TabExport() {
             </li>
             <li>
               la définition de chacune des données de l’export est accessible dans notre{" "}
-              <a
-                target="_blank"
-                href="https://tech.covoiturage.beta.gouv.fr/topic/topic-demandes-cee#topic-comparatif-v2-0-v3-x"
-              >
+              <a target="_blank" href="https://tech.covoiturage.beta.gouv.fr/topic/topic-export-de-trajets">
```

Biome (2.1.2)

[error] Avoid using target="_blank" without rel="noopener" or rel="noreferrer".

Opening external links in new tabs without rel="noopener" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noopener" attribute.


(lint/security/noBlankTarget)